go/vm/lfvm/instructions.go

```diff
@@ -753,6 +754,17 @@ func opCreate(c *context) {
 		return
 	}
 
+	if !value.IsZero() {
+		balance := [32]byte(c.context.GetBalance(c.params.Recipient))
```

Comment:
nit: this conversion is not needed
```suggestion
		balance := c.context.GetBalance(c.params.Recipient)
```